### PR TITLE
Add support for chef_zero

### PR DESCRIPTION
**Pull request by smcavallo**
Add support for chef_zero, in addition to chef_solo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ postgresql Cookbook CHANGELOG
 =============================
 This file is used to list changes made in each version of the postgresql cookbook.
 
+v4.0.7
```
Comment (tas50) [inline CHANGELOG.md]: Can you revert this bit. We'll handle the changelog. This confuses people who see it and think that 4.0.7 is pushed.

**Comment by smcavallo** [comment]
@tas50 - Changelog changes have been removed

**Comment by tas50** [comment]
@smcavallo Can you rebase this against the massive dump of changes that I landed on master?

**Comment by chewi** [comment]
What was the point of this change? It worked fine in Chef Zero before because, unlike Chef Solo, it does have somewhere to store the data, under `/tmp/kitchen/nodes`. This change breaks lots of existing Kitchen configurations.

**Pull request merged**